packages/flutter_tools/lib/src/commands/update_packages.dart

```diff
@@ -398,6 +398,8 @@ class PubspecYaml {
           // We're in a section we care about. Try to parse out the dependency:
           final PubspecDependency dependency = PubspecDependency.parse(line, filename: filename);
           if (dependency != null) { // We got one!
+            // track whether or not this a dev dependency.
```

Comment:
trivial nit: comments should start with a capital letter according to the dart style guide (which we include transitively through our style guide)